examples/draft-0-10-0/playground/src/DraftJsRichEditorExample.css

```diff
@@ -0,0 +1,76 @@
+.DraftJsPlaygroundContainer-editor {
```

Comment:
This are the styles used for the rich.html and draft website with minor tweaks to fit on the playground